### ISSUE TITLE
Genetics: Playing with a Diona's block 55 gibs the Diona

### ISSUE THREAD

**Issue by IAmBigCoat** [ParadiseSS13/Paradise#2940]
Diona is instantly booted to being a ghost.
The DNA Modifier's sprite doesn't update to reflect the lack of a person.
Ejecting the pod manually spits out all the diona's equipment.

This came as quite a shock.

**Comment by Crazylemon64** [comment]
<a href=https://github.com/ParadiseSS13/Paradise/blob/master/code/game/dna/genes/monkey.dm#L40-L42>This is the bit of the monkey gene causing problems</a>

Dionae don't have a lesser form, so into the mulcher they go.

**Comment by Plotron** [comment]
Very old news.

**Comment by TheDZD** [comment]
Err, didn't Dionae at one point turn into nymphs when their monkey block was turned on?

**Comment by IAmBigCoat** [comment]
I thought that was what would happen.
I was incorrect in a hilarious fashion, and the other geneticist fastened my head onto a monkeyman to try and clone me.

**Comment by KasparoVy** [comment]
Happens with Vox, Plasmamen and Slimes, too.

**Comment by Fox-McCloud** [comment]
>Happens with Vox, Plasmamen and Slimes, too.

Not a bug; if they don't have a primitive form, they're going to gib.

Diona gibbing, though, is a bug.

**Comment by Crazylemon64** [comment]
Why is that a bug? What should their primitive form be?

**Comment by Fox-McCloud** [comment]
Diona nymphs =p

**Comment by Crazylemon64** [comment]
That was commented out, I think - Probably could create a dummy species datum that turns the mob into a diona nymph, I'd think...

**Comment by Fox-McCloud** [comment]
Probably because nymphs used to be a monkey subtype, but then people got huffy about their behavior and how griefy they were, so they lose their hands and ability to equip anything---now all they are is a simple animal that can literally only bite people (and pee on trays+eat from them).

**Comment by VampyrBytes** [comment]
This is on my to do list :) As Fox said, it's only Diona's that's a bug

**Comment by Crazylemon64** [comment]
Progress...?

**Comment by VampyrBytes** [comment]
@Crazylemon64 I've got a plan to refactor species to a higher level (probably living). that way we won't have 
if(ishuman) do species way 
else do this way
all through the code. Obviously this is a big job, so I want to wait until (a) I've got plenty of time to get my teeth stuck into it, and (b) the organ refactor is finished as species and organs are pretty connected.
As the issue here is switching between mobs with and without species, this will get taken care off at the same time
